blocks/library/image/block.js

```diff
@@ -239,7 +302,11 @@ class ImageBlock extends Component {
 						const img = <img src={ url } alt={ alt } onClick={ this.onImageClick } />;
 
 						if ( ! isResizable || ! imageWidthWithinContainer ) {
-							return img;
+							return (
+								<div style={ { width, height } }>
```

Comment:
This changes feels a bit weird, we're adding an unnecessary extra div for me and I believe this div is not present in the "save" representation? Can we avoid this extra div? Why can't we add these styles to the "figure"?

Reply:
> This changes feels a bit weird, we're adding an unnecessary extra div for me and I believe this div is not present in the "save" representation?

Our CSS rules (namely `.wp-block-image.is-resized img { height: 100% }`) rely on the img being contained in something that has an explicit width and height. On desktop, this is true because `re-resizable` wraps the image in a div with an inline width and height. On mobile, however, we don't load `re-resizable` and so our CSS rules don't work. 

By wrapping the img with our own div on mobile, our CSS rules work on both mobile and desktop because our markup has a similar structure. 

> Why can't we add these styles to the "figure"?

We can't set a height on the figure because we do not know how tall the caption underneath the image is.